deploy.py

```diff
@@ -0,0 +1,140 @@
+import os
```

Comment:
Anything different notable here?

Reply:
Nope, same code as LLB (with changes for appropriate property files listed in `ENV_VALUES`)